addons/stock/models/stock_quant.py

```diff
@@ -553,9 +554,20 @@ def _merge_quants(self):
                         )
                    DELETE FROM stock_quant WHERE id in (SELECT unnest(to_delete_quant_ids) from dupes)
         """
+        params = []
+        quant_tuples_to_merge = self.env.context.get("_quant_tuples_to_merge")
```

Comment:
maybe add a comment stating where this context value comes from and what the expected format is.